onnx/defs/controlflow/old.cc

```diff
@@ -817,59 +778,46 @@ scan_output values of the scan construct (similar to the concatenated intermedia
 hidden-state values of RNN-like constructs). All the output tensors (state_variables as
 well as scan_output_element tensors) are required to have the same shape in each iteration
 of the loop (a restriction imposed to enable efficient memory allocation).
-
 Note that the iterated element passed to the body subgraph does not have a sequence
 axis. It will have a rank one less than the rank of the corresponding scan_input.
-
 The scan operation returns the final values of the state_variables as well as the
 scan_outputs.
-
 The optional attribute scan_input_directions specifies the direction (forward or backward)
 for each scan input. If this attribute is omitted, all sequences are scanned in the forward
 direction. A bidirectional scan may be performed by specifying the same tensor input twice
 in the scan_inputs, once with a forward direction, and once with a backward direction.
-
 The scan_output of the operation is produced by concatenating the scan_output_element
 values produced by the body in each iteration.  The optional attribute scan_output_directions
 specifies the direction in which scan_output is constructed (by appending or prepending the
 scan_output_element to scan_output in each iteration) for each scan_output. If this attribute
 is omitted, the scan_output_element is appended to the scan_output in each iteration.
-
 The optional attribute scan_input_axes specifies the axis to be scanned for each scan_input.
 If omitted, every scan_input will be scanned in axis 0. For example, if axis 0 is the
 batch axis and axis 1 is the time axis (to be scanned), specify an axis value of 1.
 Note that scanning a non-zero axis may be less efficient than scanning axis zero.
-
 The optional attribute scan_output_axes specifies the axis along which the scan_outputs
 are accumulated for each scan_output. For example, if axis 1 is the time axis (to be
 scanned) for both inputs and outputs, specify a scan_input axis and scan_output axis
 value of 1.
-
 Note that because of the ONNX restriction that only the last parameter of an operator can
 be variadic, the initial-states and scan-inputs are listed together as one input parameter.
 Similarly, the final-states and scan-outputs are listed together as one output parameter.
 The attribute num_scan_inputs indicates the number M of scan-inputs.
-
 The behavior of
-
```

Comment:
@jacky82226, can you remove the whitespace changes in your PR?

Reply:
Good catch. Thank you.